presto-tests/src/test/java/io/prestosql/execution/TestEventListenerBasic.java

```diff
@@ -107,11 +104,12 @@ protected QueryRunner createQueryRunner()
         return queryRunner;
     }
 
-    private static EventFilters buildEventFilters()
+    private EventFilters buildEventFilters()
```

Comment:
why `Ignore SplitCompleted events in TestEventListenerBasic` changes the method to non-static?

Reply:
conflic resolution glitch.